static/src/js/actions/__tests__/errors.test.js

```diff
@@ -1,24 +1,57 @@
+import configureMockStore from 'redux-mock-store'
+import thunk from 'redux-thunk'
 import { ADD_ERROR, REMOVE_ERROR } from '../../constants/actionTypes'
 import { addError, removeError } from '../errors'
 
+const mockStore = configureMockStore([thunk])
+
 beforeEach(() => {
   jest.clearAllMocks()
 })
 
 describe('addError', () => {
-  test('should create an action to update the authToken', () => {
-    const payload = {
+  test('should create an action to update the authToken', async () => {
```

Comment:
Can you add a mock for `addToast` in this test and show that it is called with the correct parameters.

Also, this isn't on you, but since you are here can you rename this test to something that makes sense, like "should call addToast". 

There should also be a test that shows the `ADD_ERROR` action is created when a banner notification is added